testsuite/features/secondary/srv_rename_hostname.feature

```diff
@@ -21,6 +21,49 @@ Feature: Reconfigure the server's hostname
     When I change the server's short hostname from hosts and hostname files
     And I run spacewalk-hostname-rename command on the server
 
+@proxy
+  Scenario: Copy the new server keys and configure the proxy
+    When I copy server's keys to the proxy
+    And I configure the proxy
+    Then I should see "proxy" via spacecmd
+    And service "salt-broker" is active on "proxy"
+
+@sle_minion
+  Scenario: Apply high state on the SUSE Minion to populate new server CA
+    When I apply highstate on "sle_minion"
+
+@ssh_minion
+  Scenario: Apply high state on the SUSE SSH Minion to populate new server CA
+    When I apply highstate on "ssh_minion"
+
+@rhlike_minion
+  Scenario: Apply high state on the Red Hat-like Minion to populate new server CA
+    When I apply highstate on "rhlike_minion"
+
+@deblike_minion
+  Scenario: Apply high state on the Debian-like Minion to populate new server CA
+    When I apply highstate on "deblike_minion"
+
+@buildhost
+  Scenario: Apply high state on the build host to populate new server CA
+    When I apply highstate on "build_host"
+
+@virthost_kvm
+  Scenario: Apply high state on the virthost to populate new server CA
+    When I apply highstate on "kvm_server"
+
+@pxeboot_minion
+  Scenario: Apply high state on the PXE boot minion to populate new server CA
+    When I apply highstate on "pxeboot_minion"
+
+@proxy
+  Scenario: Apply high state on the proxy to populate new server CA
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "States" in the content area
+    And I click on "Apply Highstate"
+    Then I should see a "Applying the highstate has been scheduled." text
+    And I wait until event "Apply highstate scheduled by admin" is completed
+
```

Comment:
This scenario might need to go before applying the highstate on the minions? 🤔

Reply:
I had it this way before. Since I had issues with running the highstate with Salt I switched to the UI steps and moved it at the end. I will do some further testing on this.

Reply:
Fixed.